package.json

```diff
@@ -17,7 +17,9 @@
     "generate": "nuxt generate",
     "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
     "precommit": "npm run lint",
-    "deploy": "nuxt generate && now dist",
+    "deploy:now": "nuxt generate && now dist",
+    "deploy:heroku": "heroku create && git push heroku master",
```

Comment:
please move `heroku create` into markdown instructions since you do not recreate the Heroku app every deploy